mithril-signer/src/runtime/state_machine.rs

```diff
@@ -167,7 +164,7 @@ impl StateMachine {
                         "→ Epoch has changed, transiting to Unregistered"
                     );
                     *state = self.transition_from_unregistered_to_unregistered(new_epoch).await?;
-                } else if let Some(epoch_settings) = self
+                } else if let Some(signer_registrations) = self
                     .runner
                     .get_epoch_settings()
```

Comment:
This function could be renamed for more readability:
```suggestion
                    .get_signer_registrations_from_aggregator()
```